spec/requests/supervisors_spec.rb

```diff
@@ -16,6 +16,44 @@
   web_mock = WebMockHelper.new(blacklist)
   web_mock.stub_network_connection
 
+  describe "GET /index" do
+    let!(:active_supervisor) { create(:supervisor, casa_org: org, active: true) }
+    let!(:inactive_supervisor) { create(:supervisor, casa_org: org, active: false) }
+
+    it "returns http status ok" do
+      sign_in admin
```

Comment:
Would you mind just defining `active_supervisor` and `inactive_supervisor` within the `it` block and avoiding the use of `let` (and modifying specs below accordingly)? It's easier to follow the specs, especially for new developers.

https://thoughtbot.com/blog/lets-not

Reply:
yeah sure, I forgot to move this after I rebase the last time. Thanks for noticing this.